### DISCUSSION TITLE
I wonder if there is any way to to insert a enlarged figure with scroll bar

### DISCUSSION THREAD

**Discussion by UniverGao** [squidfunk/mkdocs-material#6313] [Q&A]
I tried to use insert html and css code in *.md file like below:
```markdown
<head>
    <style>
    .image-container {  
        width: 500px;
        height: 500px;
        overflow: scroll;  
    }
    </style>
</head>
# title
and some text
then insert a figure:

<div class="image-container" markdown>

![](../ref-fig/power-state-manager/state-off.svg){ : style="height:1500px;max-width:5000px;" }

</div>
```
It could work in my vscode preview window, but witch could not in this theme preview in web (with Chrome). I wonder if there is a sulution? thx

**Comment by alexvoss** [comment]
There is a solution. If all you want to do it to include extra CSS, you can do this inside your `mkdocs.yml` using the [extra_css](https://www.mkdocs.org/user-guide/configuration/#extra_css) configuration. Then, create classes as needed for the content you want to style. Use [attribute lists](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown#attribute-lists) if you need to attach classes to Markdown content (so no raw HTML wrappers are needed).

If you want to include the extra CSS only in specific pages then you can create a custom page template, see the [customizing your theme](https://www.mkdocs.org/user-guide/customizing-your-theme/) section of the MkDocs documentation. I would advise against this approach, though, since page templates can change and you end up having a maintenance issue if the template you modified is changed in subsequent versions of Material for MkDocs.

**Comment by kamilkrzyskow** [comment]
Any progress @UniverGao? 
Like Alex said, using an extra css file is the right solution if you want to define a styling rule for the whole website.
However, using the `<style>` HTML tag for a single page is fine imo, but the additional `<head>` isn't, as there should be only one on the page, and it's already provided by the theme template. 
As you're using Markdown inside HTML, make sure you enable the `md_in_html` extension. Having said that, I'm not sure if attribute lists would work when wrapped by HTML.
